### PR TITLE
Customer staff

### PR DESCRIPTION
**Pull request by deepali1-aconnexion**

### CODE REVIEW

File: app/controllers/staffs_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      redirect_to staffs_path
+    else
+      respond_to do |format|
+        format.js { render partial: "staffs/form", locals: { staff: @staff }, replace: ".staff-form-container" }
+      end
+    end
+  end
+  
+  def destroy
+    @staff = Staff.new(id: params[:id], account_id: current_customer.current_account_id)
```
Comment (AlexanderUlitin) [inline app/controllers/staffs_controller.rb]: @deepali1-aconnexion does it possible to use method like `where(id: params[:id], account_id: current_customer.current_account_id).first`?

File: app/controllers/staffs_controller.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
+class StaffsController < ApplicationController
+
+  before_action :authenticate_customer!
+  
+  def index
+    @staffs = Staff.all(_account_id: current_customer.current_account_id).fetch
```
Comment (AlexanderUlitin) [inline app/controllers/staffs_controller.rb]: @deepali1-aconnexion does it possible to add belongs_to relation to Staff model and use it like `current_customer.current_account.staff.build` here?

Comment (AlexanderUlitin) [reply app/controllers/staffs_controller.rb]: I see it's possible to get collections https://github.com/scventures/sagn-customers-web/blob/master/app/controllers/service_requests_controller.rb#L9

File: app/controllers/staffs_controller.rb (hunk 1 above)
Comment (deepali1-aconnexion) [reply app/controllers/staffs_controller.rb]: @AlexanderUlitin if where method is used it will make api call twice.

Comment (AlexanderUlitin) [reply app/controllers/staffs_controller.rb]: @deepali1-aconnexion thanks for your reply, I'll merge it. Good job!

**Pull request merged**